frontend/src/components/SearchForm.js

```diff
@@ -1,42 +1,77 @@
-import { Fragment } from "react";
+import { Fragment, useState } from "react";
 import '../sass/components/searchform.scss';
 import '../sass/base/typography.scss';
 import icon from '../assets/search-icon.png';
 
-const SearchForm = () => (
+const SearchForm = () => {
+    const [radioDifficulty, setRadioDifficulty] = useState('beginner');
+
+    const onSetRadioDifficulty = (value) => {
+        setRadioDifficulty(value);
+    }
+
+    const difficultyHandler = (value) => {
+        onSetRadioDifficulty(value);
+    }
+
+    return (
     <Fragment>
-         <div class="search-form">
-            <div class="text-input-container">
-                <label for="name">Name</label>
+         <div className="search-form">
+            <div className="text-input-container">
+                <label htmlFor="name">Name</label>
                 <input type="text" id="name" name="name" placeholder="Search for an app idea or simply choose the difficulty level below..."/> 
                 <img className="search-icon" src={icon} alt="search"/>
             </div>
 
-            <div class="radio-bar-container">
-                <div class="radio-choice-container">
-                    <input type="radio" id="beginner" name="select" value="beginner" />
-                    <label for="beginner">
+            <div className="radio-bar-container">
+                <div className="radio-choice-container">
+                    <input 
+                        type="radio" 
+                        id="beginner" 
+                        name="select" 
+                        value="beginner" 
+                        checked={radioDifficulty === 'beginner'} 
+                        onClick={() => difficultyHandler('beginner')}
```

Comment:
For our mvp, this will do for now. However, later we should look into setting the `beginner, intermediate, and advanced` values dynamically rather than hard code it like this.